pubsub.go

```diff
@@ -235,6 +240,7 @@ func NewPubSub(ctx context.Context, h host.Host, rt PubSubRouter, opts ...Option
 		peerOutboundQueueSize: 32,
 		signID:                h.ID(),
 		signKey:               nil,
+		protoMatchFunc:        nil,
```

Comment:
we don't need to initialize this, the default is fine.

Reply:
Note that we do it for `signKey` to be explicit about it, as these are grouped together.